tests/data/README.md

```diff
@@ -50,6 +69,32 @@ Source: Personal photo edited with Gimp 2.10.
 |    5087 | 0xffe2 APP2 |    612 | `ICC_PROFILE......T........mntrRG chunk 1/1` |
 |         |             |        | ...                                          |
 
+### File [paris_extended_xmp.jpg](paris_extended_xmp.jpg)
+
+![](paris_extended_xmp.jpg)
+
+License: [same as libavif](https://github.com/AOMediaCodec/libavif/blob/main/LICENSE)
+
+Source: Metadata was extracted from `paris_exif_xmp_icc.jpg` with
+`exiftool -tagsfromfile paris_exif_xmp_icc.jpg paris_exif_xmp_icc.xmp`. The text of the first book of
+[De finibus bonorum et malorum](https://en.wikipedia.org/wiki/De_finibus_bonorum_et_malorum) was manually inserted in
+that file under the tag `xmp:Label` and the second book under the tag `xmp:Nickname` (any `<` or `>` removed to avoid
+conflicts with XMP). The file was reconstructed with
+`exiftool -tagsfromfile paris_exif_xmp_icc.xmp -Exif= -icc_profile= paris_exif_xmp_icc.jpg -o paris_extended_xmp.jpg`.
+The goal is to have a large XMP blob so that it can only be stored as multiple extended XMP chunks.
+
+|  address | marker      | length | data                             |
+|---------:|-------------|-------:|----------------------------------|
+|        0 | 0xffd8 SOI  |        |                                  |
+|        2 | 0xffe0 APP0 |     16 | `JFIF.....,.,.`                  |
+|       20 | 0xffe1 APP1 |   5531 | http://ns.adobe.com/xap/1.0/.<?x |
+|     5553 | 0xffe1 APP1 |  65535 | http://ns.adobe.com/xmp/extensio |
+|    71090 | 0xffe1 APP1 |  65535 | http://ns.adobe.com/xmp/extensio |
+|   136627 | 0xffe1 APP1 |   4791 | http://ns.adobe.com/xmp/extensio |
+|          |             |        | ...                              |
+
+https://en.wikipedia.org/wiki/De_finibus_bonorum_et_malorum
```

Comment:
is this leftover since the link is given in the above paragraph?

Reply:
Good catch.